src/components/NavBar/NavBar.js

```diff
@@ -17,17 +20,30 @@ import {
 } from 'react-icons/fa';
 import checkLoginStatus from '../../redux/thunks/navLoginThunk';
 import logo from '../../images/logo-no-background.png';
+import logout from '../../redux/thunks/navLogoutThunk';
 
 function NavBar() {
+  const [windowDimensions, setWindowDimensions] = useState(null);
+  useEffect(() => {
+    setWindowDimensions(window.innerWidth);
+  }, []);
+  useEffect(() => {
+    function handleResize() {
+      setWindowDimensions(window.innerWidth);
+    }
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, []);
+  const isMobile = windowDimensions <= 768;
   const dispatch = useDispatch();
-  const statusLogin = localStorage.getItem('token');
+  const auth = localStorage.getItem('token');
   useEffect(() => {
     dispatch(checkLoginStatus());
-  }, [dispatch]);
+  }, [auth]);
```

Comment:
Optional:

"auth" is a little unclear, don't you think that a name like isAuthenticated, or isLoggedIn, or userLogged, would be clearer to understand?